server/routes/apps.js

```diff
@@ -1,13 +1,15 @@
 import express from 'express';
-import appsController from '../controller/AppsController';
+import AppsController from '../controller/AppsController';
 
 const router = express.Router();
 
 /* GET home page. */
 
-router.get('/get/all', appsController.getAllApp);
-router.post('/create', appsController.createApp);
+router.get('/apps', AppsController.getApps);
+router.get('/apps/:appName', AppsController.getEventDetails);
+router.get('/apps/:appName/authorizedAccounts', AppsController.getAuthorizedAccounts);
+
 module.exports = {
-  path: '/app',
+  path: '/api/v1',
```

Comment:
assuming you are gonna use `path` to append to route so you can call something like this
/api/v1/your-path

But this way you are taking away all the benefits. versioning is there for backward compatibility. Once you rename the variable to `/api/v2` all previous routes are gone. 

Correct me if I made a wrong assumption.